application/src/main/java/org/opentripplanner/transit/service/TransitService.java

```diff
@@ -169,6 +169,13 @@ public interface TransitService {
    */
   Collection<Trip> getAllTrips();
 
+  /**
+   * Get canceled trips.
+   *
+   * @param feeds If not null, used for filtering.
+   */
+  Collection<TripOnServiceDate> getCanceledTrips(@Nullable List<String> feeds);
```

Comment:
Is there a difference between null and the empty list?

Reply:
And according to the naming convention this should be called findCancelledTrips.

There could also be a version of this called listCancelledTrips which you can call when you want all of them.